Src/BootCamp.Chapter/FileLogger.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.IO;
+
+namespace BootCamp.Chapter
+{
+    public class FileLogger : ILogger
+    {
+        private readonly string logFile = $"Log_{DateTime.Now.Day}-{DateTime.Now.Month}-{DateTime.Now.Year}.log";
+        private string _message;
+
+        public void SetMessage(string message)
+        {
+            _message = message;
+        }
+
+        public FileLogger()
+        {
+        }
+
+        public FileLogger(string message)
+        {
+            _message = message;
+        }
+
+        public void Log()
```

Comment:
Can be removed, message should be passed straight to Log method.

Reply:
Changed in [5810276](https://github.com/csinn/CSharp-From-Zero-To-Hero/pull/215/commits/58102760f5e95c0fde15237a0be1be8bf5399646)